src/schemas.py

```
@@ -41,6 +70,38 @@ class WorkspaceGet(WorkspaceBase):
+    )
+
+    @model_validator(mode="after")
+    def validate_summary_thresholds(self) -> Self:
+        """Validate that short summary threshold <= long summary threshold."""
+        short = self.messages_per_short_summary
+        long = self.messages_per_long_summary
+
+        if short is not None and long is not None and short > long:
+            raise ValueError(
+                "messages_per_short_summary must be less than messages_per_long_summary"
+            )
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)